flow-typed/npm/babel-plugin-transform-react-jsx-source_vx.x.x.js

```diff
@@ -0,0 +1,32 @@
+// flow-typed signature: 94a58ea1333bada1e79278664744bcec
```

Comment:
Should this file be in Git repo? Or should we ignore it?

Reply:
I don't know this stuff is autogenerated I think that flow-typed fetch the types from npm and probably tracks this modules

Reply:
Yeah, I checked the [documentation](https://github.com/flowtype/flow-typed/wiki/FAQs#why-do-i-need-to-commit-the-libdefs-that-flow-typed-installs-for-my-project) and it should be included in Git. It's kind of additional overhead, but whatev.